README.md

```diff
@@ -25,6 +25,9 @@ These are vulnerable applications! 🔥
 - A9 - Using Components With Known Vulnerabilities - [Cimentech](owasp-top10-2017-apps/a9/cimentech)
 - A10 - Insufficient Logging&Monitoring - [GamesIrados.com](owasp-top10-2017-apps/a10/games-irados)
 
+## Want to add your contribution?
+Be sure to check [this](/docs/Contribute.md) section!
```

Comment:
Suggestion: `We encourage you to contribute to SecDevLabs! Please check out the Contributing to SecDevLabs guide for guidelines about how to proceed.`

Also, it would be better if the name of the file were `CONTRIBUTING.md`.

Reply:
Thanks @rafaveira3 ! I liked that! I'll change it here!